### PR TITLE
[4.2] Remove useless code

### PR DESCRIPTION
**Pull request by joomdonation**
Pull Request for Issue # .

### Summary of Changes
The code which is removed in this PR suppose to index data for every custom fields automatically. However, the code is not working at all. Since PR https://github.com/joomla/joomla-cms/pull/36747 was accepted, we can now choose to index the fields we want and how each field will be indexed, this code should be removed and should speed up the index process (as mentioned, it never worked, so it is useless). See my question https://github.com/joomla/joomla-cms/pull/36747#issuecomment-1133662687 and Hannes's answer https://github.com/joomla/joomla-cms/pull/36747#issuecomment-1133844965

### Testing Instructions
Honestly, I don't know how it should be tested. Maybe you just need to apply patch, try to run index in smart search and make sure it is still working.

### CODE REVIEW

File: plugins/system/fields/fields.php [hunk 1]
```diff
@@ -501,66 +501,7 @@ public function onContentPrepare($context, $item)
...
-						$item->{$field->name} = $field->value;
-					}
-				}
-			}
-		}
-
-		return true;
-	}
-
+
```
Comment (richard67) [inline plugins/system/fields/fields.php]: It seems that line 504 contains some spaces, so PHPCS fails in drone.

File: plugins/system/fields/fields.php [hunk 2]
```diff
@@ -502,65 +502,6 @@ public function onContentPrepare($context, $item)
...
-
-				// Getting the fields for the constructed context
-				$fields = FieldsHelper::getFields($component . '.' . $section, $tmp, true);
-
-				if (is_array($fields))
-				{
-					foreach ($fields as $field)
-					{
-						// Adding the instructions how to handle the text
-						$item->addInstruction(Indexer::TEXT_CONTEXT, $field->name);
```
Comment (Quy) [inline plugins/system/fields/fields.php:552]: Please delete.

https://github.com/joomla/joomla-cms/blob/a9ffaec4f5c0e58a2001fa0345c748deade84bb1/plugins/system/fields/fields.php#L17

Comment (joomdonation) [reply plugins/system/fields/fields.php:552]: Yes, removed. Thanks @Quy

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on c3a80a03db66fc54097c48a2816c63067c9aa30a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37852">issues.joomla.org/tracker/joomla-cms/37852</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on c3a80a03db66fc54097c48a2816c63067c9aa30a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37852">issues.joomla.org/tracker/joomla-cms/37852</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37852">issues.joomla.org/tracker/joomla-cms/37852</a>.</sub>

**Comment by roland-d** [comment]
Thanks everybody

**Pull request merged**